play.pokemonshowdown.com/src/panel-chat.tsx

```diff
@@ -135,6 +135,35 @@ export class ChatRoom extends PSRoom {
 			this.challenged = null;
 			this.update(null);
 			return false;
+		} case 'ignore': {
+			let ignore = PS.prefs.ignore || {};
+			let serverIgnore = ignore[PS.server.id] || {};
+			if (!target) return true;
+			if (toID(target) === PS.user.userid) {
+				this.receiveLine([`You are not able to ignore yourself.`]);
+			} else if (serverIgnore[toID(target)]) {
+				this.receiveLine([`c`, "", `User '` + target + `' is already on your ignore list.` +
+				`(Moderator messages will not be ignored.)`]);
```

Comment:
```suggestion
				this.receiveLine(["", `User '` + target + `' is already on your ignore list.` +
				`(Moderator messages will not be ignored.)`]);
```
This is the standard way to do unformatted text. No `c`.